cmd/server-main.go

```diff
@@ -115,7 +115,7 @@ EXAMPLES:
   5. Start distributed minio server on an 8 node setup with 8 drives each. Run following command on all the 8 nodes.
```

Comment:
Need to change this to 32 / 32 to coincide with new example

Reply:
Done fixed